_posts/2019-07-22-cfp-for-barcelona.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Call for Proposals is open!"
+authors:
+  - badboy
+socialImageSrc: "/assets/posts/barcelona/cfp-opens.png"
+---
+
+## CfP for talks
+
+Starting today the [Call for Proposals](https://cfp.rustfest.eu/events/rustfest-barcelona-2019) is open!
+
+We would love to hear your proposal for a talk. Make sure to submit before **August 19, 23:59 UTC**!
+As always, we'll be prioritizing talks not yet recorded at Rust conferences.
+
+You are free to talk about anything Rust-related, come and share your own personal Rust experience!
+From code, to community and not forgetting experiences in production, best practices, theory or anything in between. We are happy to hear it from you.
+
+First time speaking? Submit! It's totally okay! If we accept your talk in our (blind) selection procedure we're happy to be your first experience. If you'd like, we're also happy to mentor you to give you some tips.
```

Comment:
```suggestion
First time speaking? Submit! It's totally okay! If we accept your talk in our (blind) selection procedure we're happy to be your first experience. If you'd like, we're also happy to mentor you to give you some tips.
```
First time speaking? Submit! It's totally okay! If your talk will be selected in our (blind) selection procedure, we'll be happy for Rustfest to be your first experience on stage. We're also happy to mentor you with tips, guidelines and review of your talk.

Reply:
I would also evaluate a more clear wording for "stage newcomers" to review their talk. We might want every talk to be as refined and shine as possible